src/Windows/R/Components/Test/Threading/JoinableTaskContextProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.ComponentModel.Composition;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.UnitTests.Core.Threading;
+using Microsoft.VisualStudio.Threading;
+
+namespace Microsoft.R.Components.Test.Threading {
+    [Export]
+    [ExcludeFromCodeCoverage]
+    public sealed class JoinableTaskContextProvider {
```

Comment:
This is the lowest level of test assemblies that a) exports components and b) has dependencies on core text editor. Text editor needs this export otherwise view/buffer creation fails.